### PR TITLE
Add java in subcategory of Languages

### PR DESCRIPTION
**Pull request by devWonderkind**
## Fixes Issue
#Closes 1051 

## Changes proposed

Add Subcategory Java in "Language" category
Add resource in Subcategory java of "Language"

## Screenshots

linkshb
 

## Note to reviewers

@CBID2 thanks for assigning the task. 
Please merge the PR.🙃

### CODE REVIEW

File: database/languages/java.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[
+    {
+        "name": "Oracle java Tutorial",
+        "description": "This is the official Java tutorial provided by Oracle, the company behind Java. It covers the basics of Java programming and provides comprehensive documentation on various Java topics.",
```
Comment (CBID2) [inline database/languages/java.json:4]: [Proofreading]
```suggestion
        "description": "This is the official Java tutorial provided by Oracle, the company behind Java. It covers the basics of this programming language and provides comprehensive documentation on various Java topics.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Just need some improvement

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

**Review by CBID2 — APPROVED** [review]
LGTM! :)

**Pull request merged**